appveyor.yml

```diff
@@ -38,67 +72,113 @@ install:
 
   # Check that we have the expected version and architecture for Python
   - python --version
-  - python -c "import struct; print(struct.calcsize('P') * 8)"
+  - >-
+     python -c "import sys,platform,struct;
+     print(sys.platform, platform.machine(), struct.calcsize('P') * 8, )"
+  # Get needed submodules
+  - git submodule update --init
+
+  # Download and install static "openblas.a" to PYTHON\lib
+  - ps: |
+      $PYTHON_ARCH = $env:PYTHON_ARCH
+      $PYTHON = $env:PYTHON
+      If ($PYTHON_ARCH -eq 32) {
+          $OPENBLAS = $env:OPENBLAS_32
+          $OPENBLAS_SHA256 = $env:OPENBLAS_32_SHA256
+      } Else {
+          $OPENBLAS = $env:OPENBLAS_64
+          $OPENBLAS_SHA256 = $env:OPENBLAS_64_SHA256
+      }
+      $clnt = new-object System.Net.WebClient
+      $file = "$(New-TemporaryFile).zip"
+      $tmpdir = New-TemporaryFile | %{ rm $_; mkdir $_ }
+      $destination = "$PYTHON\lib\openblas.a"
+      echo $file
+      echo $tmpdir
+      echo $OPENBLAS
+      $clnt.DownloadFile($OPENBLAS,$file)
+      $downloaded_hash = Get-FileHash -Algorithm SHA256 $file
+      if ($downloaded_hash.hash -ne $OPENBLAS_SHA256) {
+          $downloaded_hash | Format-List
+          echo $OPENBLAS_SHA256
+          throw "Downloaded OPENBLAS zip SHA256 does not match."
+      }
+      Expand-Archive $file $tmpdir      
+      rm $tmpdir\$PYTHON_ARCH\lib\*.dll.a
+      $lib = ls $tmpdir\$PYTHON_ARCH\lib\*.a | ForEach { ls $_ } | Select-Object -first 1
+      echo $lib
+      cp $lib $destination
+      ls $destination
+  # Upgrade to the latest pip and setuptools.
+  - python -m pip install -U pip setuptools
 
-  # Install requirements
-  - pip install -r windows-wheel-builder\requirements.txt
-  - python -m pip install --upgrade pip
   # Pin wheel to 0.26 to avoid Windows ABI tag for built wheel
-  - pip install wheel==0.26
+  - pip install "wheel==0.26"
+
+  # Install build requirements.
   - pip install "%CYTHON_BUILD_DEP%"
-  # Download built openblas libraries
-  - cmd /c windows-wheel-builder\unpack_openblas.bat
 
 build_script:
-  # Build and install the wheel
   - cd numpy
   - git checkout %BUILD_COMMIT%
-  - python ..\windows-wheel-builder\build_numpy.py . %PYTHON_ARCH% %OPENBLAS_ROOT%
-  - cd dist
-  - pip install --pre --no-index -f . numpy
-  - pip install nose
+  - ps: |
+      $PYTHON_ARCH = $env:PYTHON_ARCH
+      If ($PYTHON_ARCH -eq 32) {
+          $MINGW = $env:MINGW_32 
+      } Else {
+          $MINGW = $env:MINGW_64
+      }
+      $env:Path += ";$MINGW"
+      $env:NPY_NUM_BUILD_JOBS = "4"
+  - python setup.py bdist_wheel
+  - ps: |
+      # Upload artifact to Appveyor immediately after build
+      ls dist -r | Foreach-Object {
+          appveyor PushArtifact $_.FullName
+          pip install $_.FullName
+      }
 
 test_script:
-  # Run the project tests
-  - python -c "import numpy; numpy.test(verbose=3)"
+  - pip install nose
   - cd ..
+  - python -c "import numpy; numpy.test(verbose=3)"
```

Comment:
The `on_success:` script needs `cd numpy` in beginning since it's looking for wheels in `$PWD/dist/`

Reply:
Good catch.